toolkit/docs/building/building.md

```diff
@@ -809,6 +809,7 @@ To reproduce an ISO build, run the same make invocation as before, but set:
 | Variable                      | Default                                                                                                | Description
 |:------------------------------|:-------------------------------------------------------------------------------------------------------|:---
 | LOG_LEVEL                     | info                                                                                                   | Console log level for go tools (`panic, fatal, error, warn, info, debug, trace`)
+| LOG_COLOR                     | auto                                                                                                   | Console log color for go tools (`always, auto, never`). `always` and `auto` enable color, `never` disables color in terminal output.
```

Comment:
Please explain what's the difference between `auto` and `always`. Right now it looks like they do the same thing. If that's the case, I think we can drop one and go with a simple `y` and `n`. In other words: let's clarify this option.:)

Reply:
I chose 'auto, always, never' to keep it consistent with the linux standard.
I can change 'always' to enable color in logs also. Are we ok with coloring the logs? 'auto' and 'never' would remain the same.
The other option is to use y/n.

Reply:
Chatted with Daniel offline, going ahead with these options:
'never' to never use colorized output (neither logs nor terminal)
'always' to always use colors (both logs and terminal)
'auto' to color terminal output (not logs, only terminal)